PowerPointLabs/PowerPointLabs/ActionFramework/Content/EffectsLabBlurrinessContentHandler.cs

```diff
@@ -11,20 +13,27 @@ class EffectsLabBlurrinessContentHandler : ContentHandler
     {
         protected override string GetContent(string ribbonId)
         {
-            var feature = ribbonId.Replace(TextCollection.DynamicMenuId, "");
+            string feature = ribbonId.Replace(TextCollection.DynamicMenuId, "");
 
-            var xmlString = new System.Text.StringBuilder();
+            System.Text.StringBuilder xmlString = new System.Text.StringBuilder();
 
             for (int i = 0; i < 7; i++)
             {
-                xmlString.Append(string.Format(TextCollection.DynamicMenuXmlButton, feature + TextCollection.DynamicMenuOptionId + (i + 4) + "0",
+                xmlString.Append(string.Format(TextCollection.DynamicMenuXmlButton, 
+                    feature + TextCollection.DynamicMenuOptionId + (i + 4) + "0",
```

Comment:
What do these numbers mean? Can you remove all the magic numbers?
Why not use ((i + 4)  * 10? Or increment by 10 in the loop?